tests/e2e/test_operator.py

```diff
@@ -2163,14 +2163,13 @@ def test_032(self):
         timeout=int(1000),
         )
 
-
     note("Setting the thread event to true...")
     trigger_event.set()
+    note("Joining the parallel thread with main...")
     join()
 
-    kubectl.launch('delete clickhouse-test-032-client')
     kubectl.delete_chi(chi)
-
+    kubectl.launch(f"delete pod clickhouse-test-032-client", ns=kubectl.namespace, timeout=600, ok_to_fail=True)
```

Comment:
ok_to_fail=False shall be here client just a  simple pod without external dependencies and shall delete without error

Reply:
Ok, change is commit now